mmdeploy/codebase/mmdet/models/detectors/panoptic_two_stage_segmentor.py

```diff
@@ -0,0 +1,41 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+from mmdeploy.core import FUNCTION_REWRITER
+
+
+@FUNCTION_REWRITER.register_rewriter(
+    'mmdet.models.detectors.panoptic_two_stage_segmentor'
+    '.TwoStagePanopticSegmentor.simple_test')
+def two_stage_panoptic_segmentor__simple_test(ctx,
```

Comment:
Could add unit test for this rewriting: https://github.com/open-mmlab/mmdeploy/blob/master/tests/test_codebase/test_mmdet/test_mmdet_models.py

Reply:
Thanks. Should I implement `test_two_stage_panoptic_segmentor` (for above)? or `test_base_semantic_head` (for another re-written `seg_heads.base_semantic_head.BaseSemanticHead`) or both?